README.md

```diff
@@ -70,7 +70,7 @@ Amplify requires the following Xcode versions, according to the targeted platfor
 | macOS         | 15.0+         |
 | tvOS          | 15.0+         |
 | watchOS       | 15.0+         |
-| visionOS      | 15 beta 2+    |
```

Comment:
Since now all platforms have the same Xcode requirement, would it be easier to just drop the table?